Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -745,6 +747,37 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     }
 }
 
+//MARK: Universal Bridge support
+extension MiniAppScriptMessageHandler {
+    func sendJsonToHostApp(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.jsonInfo else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: MiniAppJavaScriptError.invalidPermissionType.rawValue)
+            return
+        }
+        if !requestParamValue.content.isEmpty {
+            let info = MiniAppBridgeContent(jsonStringContent: requestParamValue.content)
+            self.hostAppMessageDelegate?.sendJsonToHostApp(info: info, completionHandler: { (result) in
+                self.manageJsonStringToHostAppResult(result, with: callbackId)
+            })
+        } else {
+            self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.valueIsEmpty))
+        }
+    }
+    
+    func manageJsonStringToHostAppResult(_ result: Result<MASDKProtocolResponse, Error>, with callbackId: String) {
+        switch result {
+        case .success:
+            self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: "SUCCESS")
+        case .failure(let error):
+            if !error.localizedDescription.isEmpty {
+                self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: error.localizedDescription)
```

Comment:
```suggestion
                self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: prepareMAJavascriptError(error))
```

Reply:
This is not working as the object `error` is of type `Error` and does not confirm to the `MiniAppsErrorProtocol`. 
For this I have now created new error enum object `UniversalBridgeError` to cater for universal bridge related errors. With this I can now use the `prepareMAJavascriptError(error)`